### PR TITLE
feat(connectivity_plus): Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by charafau**
ref: https://github.com/fluttercommunity/plus_plugins/issues/3152

Another spm support

removed obsolete objc code, moved plugin to swift only, which avoid mixing swift project

### CODE REVIEW

File: packages/connectivity_plus/connectivity_plus/darwin/Classes/ConnectivityPlusPlugin.m
Comment (miquelbeltran) [inline packages/connectivity_plus/connectivity_plus/darwin/Classes/ConnectivityPlusPlugin.m:1]: Would removing this code cause an issue on older apps still using Obj-C?

File: packages/connectivity_plus/connectivity_plus/example/pubspec.yaml
Comment (miquelbeltran) [inline packages/connectivity_plus/connectivity_plus/example/pubspec.yaml:1]: Changes in this file should be reverted

**Review by miquelbeltran — CHANGES_REQUESTED** [review]
The example pubspec should not be changed.

Instead, use `melos bootstrap` as explained in the contributors document to setup the example project.

File: packages/connectivity_plus/connectivity_plus/darwin/Classes/ConnectivityPlusPlugin.m
Comment (charafau) [reply packages/connectivity_plus/connectivity_plus/darwin/Classes/ConnectivityPlusPlugin.m:1]: I'm sure on 99% it won't. Example project works without changing anything, support for swift in plugin was here already.

**Comment by charafau** [comment]
> The example pubspec should not be changed.
> 
> Instead, use `melos bootstrap` as explained in the contributors document to setup the example project.


Thank you Miguel, will educate myself with melon :)

**Review by vbuberen — APPROVED** [review]
Tested and everything works good. 

Thanks for another contribution!

**Pull request merged**